app/controllers/discussions_controller.rb

```diff
@@ -4,7 +4,7 @@ class DiscussionsController < GroupBaseController
   before_filter :we_dont_serve_images_here_google_bot
   before_filter :authenticate_user!, :except => [:show, :index]
   before_filter :load_resource_by_key, except: [:new, :create, :index, :update_version]
-  authorize_resource :except => [:new, :create, :index, :add_comment]
+  authorize_resource :except => [:new, :create, :index, :add_comment, :print]
```

Comment:
Maybe something for the next iteration: would be cool if the pie chart made
sense in grayscale.

On Wed, May 6, 2015 at 10:45 AM Robert Guthrie <notifications@github.com>
wrote:

> In app/controllers/discussions_controller.rb
> <https://github.com/loomio/loomio/pull/2140#discussion_r29721347>:
>
> > @@ -4,7 +4,7 @@ class DiscussionsController < GroupBaseController
> >    before_filter :we_dont_serve_images_here_google_bot
> >    before_filter :authenticate_user!, :except => [:show, :index]
> >    before_filter :load_resource_by_key, except: [:new, :create, :index, :update_version]
> > -  authorize_resource :except => [:new, :create, :index, :add_comment]
> > +  authorize_resource :except => [:new, :create, :index, :add_comment, :print]
>
> Woha!
> Big red alert!
>
> You must authorise the resource the same way that the show action does.
> Don't merge this until this is fixed. Cheers!
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/loomio/loomio/pull/2140/files#r29721347>.
>